apps/openssl-vms.cnf

```diff
@@ -16,6 +16,10 @@ HOME			= .
  # Use this in order to automatically load providers.
 openssl_conf = openssl_init
 
+# Comment this out if you deliberately want to ignore
+# the configuration errors
```

Comment:
Also remove "the", i.e. "ignore configuration errors"

Reply:
Done. Thanks!